shogun-lib/src/main/java/de/terrestris/shoguncore/graphql/scalar/GeometryScalar.java

```diff
@@ -0,0 +1,80 @@
+package de.terrestris.shoguncore.graphql.scalar;
+
+import com.bedatadriven.jackson.datatype.jts.JtsModule;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import graphql.language.StringValue;
+import graphql.schema.*;
+import org.locationtech.jts.geom.Geometry;
+
+import java.util.HashMap;
+
+public class GeometryScalar {
+
+    public static final GraphQLScalarType GEOMETRY = new GraphQLScalarType("Geometry", "A custom scalar that handles geometries", new Coercing() {
+        @Override
+        public Object serialize(Object dataFetcherResult) {
+            return serializeGeometry(dataFetcherResult);
+        }
+
+        @Override
+        public Object parseValue(Object input) {
+            return parseGeoemtryFromVariable(input);
+        }
+
+        @Override
+        public Object parseLiteral(Object input) {
+            return parseGeometryFromAstLiteral(input);
+        }
+    });
+
+    private static boolean isAGeometry(Object possibleGeometryValue) {
+        return possibleGeometryValue instanceof Geometry;
+    }
+
+    private static Object serializeGeometry(Object dataFetcherResult) {
+        if (isAGeometry(dataFetcherResult)) {
+            Geometry geometry = (Geometry) dataFetcherResult;
+            ObjectMapper mapper = new ObjectMapper();
+            mapper.registerModule(new JtsModule());
+            try {
+                return mapper.readValue(mapper.writeValueAsString(geometry), HashMap.class);
+            } catch (JsonProcessingException e) {
+                e.printStackTrace();
```

Comment:
Please make use of the `Logger` here or let the method throw a `JsonProcessingException`.